tests/loan-edge-cases.unit.php

```
@@ -0,0 +1,665 @@
+            continue;
+        }
+        [$k, $v] = explode('=', $line, 2);
+        $k = trim($k);
+        $v = trim($v);
+        if (strlen($v) >= 2 && ($v[0] === '"' || $v[0] === "'") && $v[-1] === $v[0]) {
+            $v = substr($v, 1, -1);
+        }
+        $env[$k] = $v;
+    }
+    return $env;
+}
```

PHPMD (2.15.0)

[error] Remove error control operator '@' on line 39. (undefined)

(ErrorControlOperator)